backend/layers/common/entities.py

```diff
@@ -172,6 +172,7 @@ class DatasetMetadata:
     development_stage: List[OntologyTermId]
     cell_type: List[OntologyTermId]
     cell_count: int
+    primary_cell_count: int
```

Comment:
Last thing to support the hybrid state where this isn't backfilled--I believe this type hint needs to be wrapped as Optional[int] and set to default to = None (i.e. `primary_cell_count: Optional[int] = None`)